docs/releases/3.1.txt

```diff
@@ -96,6 +96,15 @@ and generate and apply a database migration. For now, the old fields and
 transforms are left as a reference to the new ones and are :ref:`deprecated as
 of this release <deprecated-jsonfield>`.
 
+``DEFAULT_HASHING_ALGORITHM`` settings
+--------------------------------------
+
+The new :setting:`DEFAULT_HASHING_ALGORITHM` setting allows defining the
+default hashing algorithm to use for encoding cookies, password reset tokens in
+the admin site, user sessions, and signatures created by
+:class:`django.core.signing.Signer` and :meth:`django.core.signing.dumps`.
+Algorithm must be supported by :py:mod:`hashlib`.
+
```

Comment:
Can we include a paragraph about how to upgrade? Something along these lines:

> When you're upgrading to Django 3.1 ensure that all your servers are running Django 3.1 before settings this value to `'sha256'` which is the recommended value. `'sha256'` will also be the default value from Django 4.0.

Reply:
See my comment above & I'll rather be "secure" by default. I don't want to push changing the default out to the year 2022.

Reply:
Thanks @felixxm. Looks good! :+1: